[BE] fix: QA에서 발생한 오류를 수정한다.

donghoony opened the pull request:
<!-- 제목: [BE/FE/All] (기능 등) -->
<!-- 아래에 이슈 번호를 매겨주세요 -->

- resolves #450 

---

### 🚀 어떤 기능을 구현했나요 ?
- see #450 

### 🔥 어떻게 해결했나요 ?
- 각각을 간단하게 해결했어요

donghoony committed:
fix: 카테고리 개수 2개로 수정

donghoony committed:
fix: 리뷰이 이름 치환

donghoony committed:
style: apply code convention

Kimprodp reviewed (approved):
굿뜨

nayonsoso reviewed (approved):
구구구굳

![image](https://github.com/user-attachments/assets/e53e77dd-441b-4bf9-b5ce-fd624579b69a)

donghoony committed:
fix: 필요하지 않은 문구 제거

hyeonji1220 reviewed (approved):
므지

donghoony committed:
chore: resolve conflict

Merge remote-tracking branch 'refs/remotes/origin/develop' into be/fix/450-fix-qa

# Conflicts:
#	backend/src/main/java/reviewme/review/service/ReviewDetailLookupService.java
#	backend/src/main/java/reviewme/reviewgroup/repository/ReviewGroupRepository.java
